rafs/src/metadata/md_v6.rs

```diff
@@ -48,7 +48,7 @@ impl RafsSuper {
 
         self.meta.flags = RafsSuperFlags::from_bits(ext_sb.flags())
             .ok_or_else(|| einval!(format!("invalid RAFS flags 0x{:x}", ext_sb.flags())))?;
-        info!("RAFS features: {}", self.meta.flags);
+        debug!("RAFS features: {}", self.meta.flags);
```

Comment:
It seems useful when we use `nydus-image check`.

Reply:
> It seems useful when we use `nydus-image check`.

I think so.

Reply:
It generates annoying message when running containers:
root@liujiang-ecs-dev:~/ws/nydus-snapshotter.git/cmd/containerd-nydus-grpc# nerdctl run -ti --rm --snapshotter nydus nginx
docker.io/library/nginx:latest:                                                   resolved       |++++++++++++++++++++++++++++++++++++++| 
index-sha256:593dac25b7733ffb7afe1a72649a43e574778bf025ad60514ef40f6b5d606247:    done           |++++++++++++++++++++++++++++++++++++++| 
manifest-sha256:d2b2f2980e9ccc570e5726b56b54580f23a018b7b7314c9eaff7e5e479c78657: done           |++++++++++++++++++++++++++++++++++++++| 
config-sha256:eb4a57159180767450cb8426e6367f11b999653d8f185b5e3b78a9ca30c2c31d:   done           |++++++++++++++++++++++++++++++++++++++| 
elapsed: 42.3s                                                                    total:  11.5 K (278.0 B/s)                                       
[2023-06-29 21:43:19.647693 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.828587 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.920714 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921195 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921417 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921645 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921847 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh